src/test/completion.test.ts

```diff
@@ -74,7 +74,7 @@ suite('JSON Completion', () => {
 		const position = Position.create(0, offset);
 		const jsonDoc = ls.parseJSONDocument(document);
 		return ls.doComplete(document, position, jsonDoc).then(list => {
-			if (expected.count) {
+			if (typeof expected.count === 'number') {
```

Comment:
if you call this method with `expected.count === 0` it won't check zero length (`assert.equal(list!.items.length, expected.count`)

but after this change, it seems that test for `test('In comment', ...` is not correct